swagger.yaml

```diff
@@ -402,6 +402,98 @@ paths:
         "200":
           description: Paid status updated
 
+  "/software":
+    get:
+      summary: Get all software entries
+      description: Retrieve a list of all software entries in the registry.
+      responses:
+        '200':
+          description: A list of software entries
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/SoftwareRegistry'
+
+    post:
+      summary: Register new software
+      description: Register a new software entry in the registry.
+      requestBody:
+        description: Software entry to register
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/SoftwareRegistry'
+      responses:
+        '201':
+          description: Software entry created successfully
+        '400':
+          description: Bad request, software entry already exists or invalid data
```

Comment:
It looks like specs don't typically define specific error responses, see https://github.com/Metatavu/time-bank-api-spec/blob/develop/swagger.yaml for some examples.
Rather they seem to use "default".
Here is another good and larger example if needed:
https://github.com/Metatavu/etra-cloud-api-spec/blob/24e30f958a099d9681ebd3193c06d04c95c0bdd7/swagger.yaml

You should update these for all endpoints